internal/provisioner/objects/rbac/clusterrole/cluster_role.go

```diff
@@ -87,7 +88,7 @@ func desiredClusterRole(name string, contour *model.Contour) *rbacv1.ClusterRole
 			// Gateway API resources.
 			// Note, ReferencePolicy does not currently have a .status field so it's omitted from the status rule.
 			policyRuleFor(gatewayv1alpha2.GroupName, getListWatch, "gatewayclasses", "gateways", "httproutes", "tlsroutes", "referencepolicies"),
-			policyRuleFor(gatewayv1alpha2.GroupName, createGetUpdate, "gatewayclasses/status", "gateways/status", "httproutes/status", "tlsroutes/status"),
+			policyRuleFor(gatewayv1alpha2.GroupName, update, "gatewayclasses/status", "gateways/status", "httproutes/status", "tlsroutes/status"),
```

Comment:
This was reduced to just `update` to match https://github.com/projectcontour/contour/blob/main/internal/k8s/rbac.go#L23